### PR TITLE
 Anuvrat Parashar's talk - Notes 

### PR DESCRIPTION
**Pull request by vipulgupta2048**

### CODE REVIEW

**Comment by sourabhtk37** [comment]
Please also update the talks-log stating the last meetups talks and links if you can else just add yours and anuvrat's.

**Comment by vipulgupta2048** [comment]
Check it out. I have updated the docs. As requested.

File: talks-log/2017.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 
+- **Applications and GUI Overview** by [Raju Devidas](https://github.com/rajudev)
+([8-10-2017])
+- **Installation Demo** by [Jatin]( https://github.com/cocoa1231)
+([8-10-2017])
+- **Performing common tasks on linux** by [T K Sourabh](https://github.com/sourabhtk37)
+([8-10-2017])
+- **Basics to Shell** by [Kuntal Majumder](https://github.com/hellozee)
+([8-10-2017])
+- **Software development on linux** by [Anuvrat Parashar](https://github.com/bhanuvrat)
```
Comment (sourabhtk37) [inline talks-log/2017.md]: Link the file stored on the slides folder here.

File: slides/talk.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+###	Software development on Linux
```
Comment (sourabhtk37) [inline slides/talk.md]: Rename the file appropriately with maybe the talk title with name of speaker.

**Comment by vipulgupta2048** [comment]
Changes done.

**Comment by vipulgupta2048** [comment]
Final changes. Ready for review @sourabhtk37

**Comment by sourabhtk37** [comment]
Thanks for the PR :heart: 
Make sure next time to squash your commits.

**Pull request merged**